index.html

```diff
@@ -7,14 +7,13 @@
   </head>
 
   <body>
-
     
       <h1 id="title" style="text-align: center; text-transform: uppercase;">Microverse Survey Form</h1>
       <p style="text-align: center; text-transform: uppercase; " id="description">This Survey form is intented to collect data of all prospective students of Microverse</p>
    
       <div style="margin-top: 4%;"></div>
 
-    <form id="survey-form" action="">
+    <form id="survey-form" action="" >
```

Comment:
Optional - Kindly include an action attribute that will ensure that the form data is submitted to any destination.